index.html

```diff
@@ -226,7 +226,9 @@ <h3>
 href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
 in time. Concrete serializations for VC-JWP might be provided, if an <a
 href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. The final output specifications might not map directly to
+published in time. 
+Concrete serializations for <a href="https://github.com/w3c-ccg/lds-jws2020/">Linked Data Signatures for JWS</a> will be provided.
```

Comment:
```suggestion
Concrete serializations for <a href="https://github.com/w3c-ccg/lds-jws2020/">Data Integrity Proofs for JWS</a> will be provided.
```

Reply:
We should merge this cryptosuite in with the others in the list above. There is no need to call it out any differently than the VC-JWT, Ed25519, and Secp256k1 ones.

Reply:
If we're going to change the title associated with the link, we should use the correct one, which is "JSON Web Signature 2020".

As another point, it's not just "a JWS Cryptosuite".  It's a specific JWS cryptosuite - the one referenced by the link.

Reply:
do the adjustments in ee77c9a work?

Reply:
I was suggesting you be more general instead of locking yourself into that specific cryptosuite. If you lock yourself into that specific cryptosuite (which already has deployment) you might not be able to change things you want to. Again, helps to be a bit vague and state that you're taking an input document (and might change it significantly throughout the WGs existence). That is why all the other cryptosuites are not specific wrt. the output suite, other than it being a concrete one based on the input document.